observers/slack_resource.cpp

```diff
@@ -26,6 +26,9 @@ Try<Nothing> SlackResourceObserver::consume(const ResourceUsage& usage) {
         Result<double_t> slackResource = CalculateCpuSlack(
             (*previousSample), executor);
         if (slackResource.isSome()) {
+            LOG(INFO) << this->name << "Estimated revocable resources for "
+                      << executor.executor_info().name() << ": "
```

Comment:
Can you also add the framework id? executor id's and names are not unique across frameworks